homeassistant/components/airzone/const.py

```diff
@@ -0,0 +1,10 @@
+"""Constants for the Airzone integration."""
+
+from typing import Final
+
+DOMAIN: Final = "airzone"
+MANUFACTURER: Final = "Airzone"
+
+AIOAIRZONE_DEVICE_TIMEOUT_SEC: Final = 10
+DEFAULT_LOCAL_API_HOST: Final = ""
```

Comment:
```suggestion
```

Reply:
Changed in 750f0f54e625f9b9e18c4f5d88652a4200753d35